### DISCUSSION TITLE
How generate PropertyChanged with new Cli Tools

### DISCUSSION THREAD

**Discussion by DiegoStefanon** [linq2db/linq2db#4030] [Q&A] [answered]
How generate PropertyChanged with new Cli Tools ?

Many thanks

**Comment by MaceWindu** [comment]
That's not something we plan to add currently. LinqToDB is ORM, not a code-generation tool.

You can continue to use T4 template for it.

**Reply by MaceWindu** [reply]
We could reconcider it if you provide more details how it is used with generated data model in your case

**Comment by igor-tkachev** [comment, marked as answer]
T4 template support it. If you add NotifyPropertyChanged.ttinclude into your tt file, it will generate support for the INotifyPropertyChanged interface.
